### PR TITLE
Update metadata (Staging chains)

### PR DESCRIPTION
**Pull request by dmytrotkk**

### CODE REVIEW

**Review by payvint — APPROVED** [review]

File: metadata/mainnet/chains.json [hunk 1]
```diff
@@ -39,17 +39,17 @@
...
   },
   "haunting-devoted-deneb": {
     "alias": "Crypto Colosseum",
     "background": "#101010",
     "category": "games",
-    "description": "Become the most powerful and important warrior patron.",
+    "description": "Crypto Colosseum is a suite of strategy based blockchain games live on SKALE.",
     "url": "https://cryptocolosseum.com/"
   },
   "potable-majestic-rasalhague": {
```
Comment (TheGreatAxios) [inline metadata/mainnet/chains.json:55]: Remove this chain - no longer active

**Review by TheGreatAxios — CHANGES_REQUESTED** [review]

File: metadata/mainnet/chains.json (hunk 1 above)
Comment (TheGreatAxios) [inline metadata/mainnet/chains.json:55]: Remove this chain, no longer active

**Review by TheGreatAxios — CHANGES_REQUESTED** [review]

**Review by SKALE-jace — APPROVED** [review]

**Review by payvint — APPROVED** [review]

File: metadata/mainnet/chains.json (hunk 1 above)
Comment (dmytrotkk) [reply metadata/mainnet/chains.json]: removed

File: metadata/mainnet/chains.json [hunk 2]
```diff
@@ -39,26 +39,19 @@
...
     "url": "https://cryptocolosseum.com/"
   },
-  "potable-majestic-rasalhague": {
-    "alias": "MyLilius",
-    "background": "#ffffff",
-    "category": "apps",
-    "url": "https://www.mylilius.com/",
-    "description": "The Hub for a Multi-Network Future"
-  },
   "snappy-quarrelsome-salm": {
```
Comment (TheGreatAxios) [inline metadata/mainnet/chains.json:55]: Remove this chain, it is not active

File: metadata/mainnet/chains.json
Comment (TheGreatAxios) [inline metadata/mainnet/chains.json:1]: Remove elegant-answer 
No longer live

Please remove the chain above and add Clet Domains to the honorable-steel-rasalhague chain

Comment (TheGreatAxios) [reply metadata/mainnet/chains.json:1]: ^ Line 89

**Review by TheGreatAxios — CHANGES_REQUESTED** [review]
Please remove old chains

File: metadata/mainnet/chains.json
Comment (dmytrotkk) [reply metadata/mainnet/chains.json:1]: moved to Calypso 👍

**Pull request merged**